src/installing-workflow/configuring-registry.md

```diff
@@ -31,6 +31,9 @@ You are now ready to `helm install deis/workflow --namespace deis -f values.yaml
 [storage]: configuring-object-storage
 [ecr]: http://docs.aws.amazon.com/AmazonECR/latest/userguide/ECR_GetStarted.html
 [gcr]: https://cloud.google.com/container-registry/
+[acr]: https://docs.microsoft.com/en-us/azure/container-registry/
+[dockerhub]: https://hub.docker.com/
+[quay]: https://quay.io/plans/
```

Comment:
Since we're just referencing quay.io above would it make more sense to just link to https://quay.io instead?